sgl-kernel/csrc/cpu/gemm.h

```diff
@@ -32,8 +33,13 @@ inline int64_t get_row_size<int8_t>(int64_t K) {
   return K + sizeof(int32_t);
 }
 
-inline int64_t get_row_size(int64_t K, bool use_int8_w8a8) {
-  return use_int8_w8a8 ? K + sizeof(int32_t) : K;
+template <>
+inline int64_t get_row_size<c10::Float8_e4m3fn>(int64_t K) {
+  return K + sizeof(int32_t);
+}
+
+inline int64_t get_row_size(int64_t K, bool use_int8_w8a8_or_fp8_w8a16) {
+  return use_int8_w8a8_or_fp8_w8a16 ? K + sizeof(int32_t) : K;
```

Comment:
don't need this for fp8. we need to add K * sizeof(int32) for int8 because, for avx512-vnni we need to do compensation to change s8s8 to u8s8 in the kernels. don't have to do this for fp8